### PR TITLE
Validate if the current directory parameter

### PR DESCRIPTION
**Pull request by EHER**
The parameter should not be an empty string.

### CODE REVIEW

File: src/Console/App.php [hunk 1]
```diff
@@ -44,6 +44,7 @@ public function __invoke(InputInterface $input, OutputInterface $output): int
         Psl\invariant(is_array($exclude), '$exclude must be an array.');
         Psl\invariant(is_string($extension), '$extension must be a string.');
         Psl\invariant(is_string($currentDirectory), '$currentDirectory must be a string.');
+        Psl\invariant(!empty($currentDirectory), '$currentDirectory must NOT be an empty string.');
```
Comment (malukenho) [inline src/Console/App.php:47]: `'' !== $currentDirectory`

**Review by malukenho — CHANGES_REQUESTED** [review]

File: src/Console/App.php (hunk 1 above)
Comment (EHER) [reply src/Console/App.php]: Done

File: src/Console/App.php [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
 use DateTimeImmutable;
-use Psl;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 use function array_map;
 use function is_array;
 use function is_string;
+use function Psl\invariant;
```
Comment (malukenho) [inline src/Console/App.php:23]: Why did you import this?

Comment (EHER) [reply src/Console/App.php]: Codana is [complaining about invariant](https://github.com/codelicia/depreday/pull/22/files). 

Importing doesn't help so it's back to how it was.

**Review by malukenho — APPROVED** [review]
Legit! Goed gedaan!

**Pull request merged**